lib/generators/manageiq/provider/provider_generator.rb

```diff
@@ -4,11 +4,29 @@ module ManageIQ
   class ProviderGenerator < PluginGenerator
     source_root File.expand_path('templates', __dir__)
 
+    def self.manager_types
+      @manager_types ||= {
+        "automation"    => "AutomationManager",
+        "cloud"         => "CloudManager",
+        "configuration" => "ConfigurationManager",
+        "container"     => "ContainerManager",
+        "infra"         => "InfraManager",
+        "monitoring"    => "MonitoringManager",
+        "network"       => "NetworkManager",
+        "physical"      => "PhysicalInfraManager",
+        "provisioning"  => "ProvisioningManager",
+        "storage"       => "StorageManager"
+      }
+    end
+
     class_option :vcr, :type => :boolean, :default => false,
                  :desc => "Enable VCR cassettes (Default: --no-vcr)"
 
-    class_option :dummy, :type => :boolean, :default => false,
-                 :desc => "Generate dummy implementations (Default: --no-dummy)"
+    class_option :scaffolding, :type => :boolean, :default => true,
+                 :desc => "Generate default class scaffolding (Default: true)"
```

Comment:
should the default be `true` or `scaffolding`?

Reply:
Yeah you're right, done